x-pack/legacy/plugins/security/index.js

```diff
@@ -91,8 +92,6 @@ export const security = (kibana) => new kibana.Plugin({
         secureCookies: securityPlugin.__legacyCompat.config.secureCookies,
         session: {
           tenant: server.newPlatform.setup.core.http.basePath.serverBasePath,
-          idleTimeout: securityPlugin.__legacyCompat.config.session.idleTimeout,
-          lifespan: securityPlugin.__legacyCompat.config.session.lifespan,
```

Comment:
We don't need to inject `idleTimeout` anymore (and we actually never needed to inject `lifespan`).

These injected vars were used by the browser to estimate when a user's session would expire. Now, the browser makes an API call to check session info and get the actual expiration time. So these injected vars are unused and we can remove them.